db/migrate/20170712033920_add_language_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLanguageToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :language, :string
```

Comment:
How are we allowing users to set their language preference; through query parameters? I think if we want to store it for a user we should consider doing it in e.g. a cookie that remembers a drop-down setting that allows changing between pages. In general I like trying to store as little in the database as we can. What do you think about splitting out the database user configuration stuff into a follow-up PR? There's little else here that's contentious so would be cool to get it merged first.